docs/install/ha-control-plane.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Running a highly-available Knative control plane"
+weight: 10
+type: "docs"
+---
+
+Learn how to run an highly-available (HA) control plane using leader election.
```

Comment:
replace "an" -> "a"

Reply:
I'd actually combine this and the section below, something like:

It is important to to provide a resilient, highly-available (HA) control plane in case a controller instance dies unexpectedly.
Beginning in release 0.13, Knative provides the ability to leader elect controllers.
This guide explains how you can run a HA control plane using leader election.